keyboards/frobiac/blackbowl/matrix.c

```diff
@@ -0,0 +1,173 @@
+// Copyright 2013 Oleg Kostyuk <cub.uanic@gmail.com>
+// Copyright 2017 Erin Call <hello@erincall.com>
+// Copyright 2023 @frobiac
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+// This implements a custom matrix scan for the BlackBowl keyboard.
+// Each side has a dedicated MCP202 I2C address are
+
+#include <stdint.h>
```

Comment:
This should be converted to custom matrix lite, as it significantly reduces the amount of custom code needed: 
https://docs.qmk.fm/#/custom_matrix?id=lite

Reply:
Great pointer, did not see that such a cleanup was possible. Thanks for suggesting